<?xml version="1.0"?>
<launch>

<!-- scene database (MongoDB) -->
   <node name="rosplan_scene_database" pkg="mongodb_store" type="mongodb_server.py" respawn="false" output="log">
	    <param name="database_path" value="$(find rosplan_knowledge_base)/common/mongoDB" />
   </node>
   <node name="rosplan_scene_message_store" pkg="mongodb_store" type="message_store_node.py" respawn="false" output="log">
   </node>

  <node name="squirrel_fill_floor"
        pkg="squirrel_object_perception"
        type="squirrel_fill_floor"
        required="true" output="screen">
	<param name="octomap_path" value="$(find squirrel_object_perception)/data/octomaps/GH30_IROS2017_edited.bt" />
	<param name="filled_octomap_path" value="$(find squirrel_object_perception)/data/octomaps/GH30_IROS2017_filled_floor.bt" />
	<param name="min_x" value="1.7"/>
	<param name="min_y" value="-0.78"/>
	<param name="max_x" value="2.55"/>
	<param name="max_y" value="-0.16"/>
  </node>
</launch>
